src/Component/PreviewMap/PreviewMap.tsx

```diff
@@ -196,13 +204,18 @@ export const PreviewMap: React.FC<PreviewMapProps> = ({
   useEffect(() => {
     const map = mapRef.current;
     const dataLayer = dataLayerRef.current;
-    if (!data && dataLayer && map && style) {
-      const geoms = GeometryUtil.getSampleGeomFromStyle(style, map.getView().getProjection());
-      geoms.forEach((geometry: any) => {
-        const feature = new OlFeature({ geometry });
-        dataLayer.getSource().addFeature(feature);
-      });
-      zoomToData();
+    setErrorMsg(undefined);
+    try {
+      if (!data && dataLayer && map && style) {
+        const geoms = GeometryUtil.getSampleGeomFromStyle(style, map.getView().getProjection());
+        geoms.forEach((geometry: any) => {
+          const feature = new OlFeature({ geometry });
+          dataLayer.getSource().addFeature(feature);
+        });
+        zoomToData();
+      }
+    } catch (error) {
+      setErrorMsg(`Could not create sample geometries: ${error}`);
```

Comment:
Can you provide i18n for the string? Every text in the UI should be translatable.